pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 import Head from "next/head";
 import Image from "next/image";
 import { Inter } from "@next/font/google";
+
 import styles from "@/styles/Home.module.css";
 
 const inter = Inter({ subsets: ["latin"] });
```

Comment:
Take a look at the font section on Nextjs doc, you will find what is this line for